### PR TITLE
Moved api server location to env config

### PR DESCRIPTION
**Pull request by Br3nda**
the production url was hard coded in several places - this changes it to be in .env file. then we can switch between prod/staging and our own running dev instances just by changing .env.
I've also included 3 example env files, and adding some help in readme.

### CODE REVIEW

File: pages/account.js [hunk 1]
```diff
@@ -42,9 +42,10 @@ account.getInitialProps = async function({ req }) {
   }
 
   const houseId = await axios
-    .get(`https://app.wharehauora.nz/api/v1/homes?access_token=${token}`)
+    .get(`${process.env.API_BASE}/homes?access_token=${token}`)
     .then(function(res) {
-      return res.data.data[0].id;
+      return '1';
```
Comment (Br3nda) [inline pages/account.js]: oops. this was me trying to fix the npm build. I'll remove this from the pr

**Comment by Br3nda** [comment]
this is causing a bunch of errors - i can't get dotenv module working, it doesn't seem to be reading .env files.
Is it reading those files for the other devs who've worked on it?

**Comment by MeghanHughes** [comment]
hmmm - no I don't think so? I will check in with @iainkirkpatrick tomorrow and see if he can shed light on it.

**Comment by Br3nda** [comment]
It's a bit of a mystery - I spent most of my lunch break trying to work out why the variables don't show up in the browser. 

Don't sink time into it - we can keep the hardcoded stuff. I can re-open this if i get it working.

**Pull request closed without merging**